Packs/ArcherRSA/Integrations/ArcherV2/ArcherV2.yml

```diff
@@ -66,6 +66,12 @@ configuration:
   name: proxy
   required: false
   type: 8
+- display: API Endpoint
+  hidden: false
+  name: api_endpoint
+  required: false
+  defaultvalue: api
+  type: 0
```

Comment:
* move it up to be right after the `Server URL` parameter (as they belong together).
* change the default value to `rsaarcher/api`
* in the code, make sure it doesn't create a duplicate in the URL (`<base URL>/rsaarcher/rsaarcher/api`)
* remove the `rsaarcher` suffix from the `Server URL` example.